cpmonitor/templates/base.html

```diff
@@ -60,7 +59,7 @@
                href="{% url 'jetzt-spenden' %}"
                type="button">Jetzt spenden</a>
           </div>
-          <div id="nav-burger" class="nav-burger" onclick="toggleMobile()">
```

Comment:
I only noticed the problem because I opened the console in the browser. I think we should write a test for this special behavior. @mdrie can you add a ticket for this? (might be a nice "good first issue" ticket :wink: )

Reply:
I do not get it. What was the problem? Not sure, what to put into that issue.

Reply:
The ticket would be:
Write an e2e test for the burger menu on mobile devices. 
The problem was just that we have special code for that, but no test that breaks when we mess with that code.

Reply:
#277